components/ProjectDetails/index.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import StyledLink from "../Layout/StyledLinkButton";
+
+export default function ProjectDetails({ projects, slug }) {
+  const result = projects.filter((project) => project.slug === slug);
+
+  if (result.length === 0) {
+    return <p>Project not found</p>;
+  }
+
+  return (
+    <>
+      <h2>Detail Page</h2>
+      <h3>{result[0].title}</h3>
```

Comment:
You can destructure the object from the array returned by your filter method in line 5 like so

```js
const objectInArray = [{ key: "value", otherKey: 10 }];

const [object] = objectInArray;

console.log(object.key);
// returns "value"
```

If you only ever expect to work with one object, this approach should be future-proof.

Ooooor you can use [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of `filter`.

Either way, no need to write `[0]`a billion times. :D